addons/project/tests/test_project_base.py

```diff
@@ -199,6 +199,7 @@ def test_change_project_or_partner_company(self):
             self.project_pigs.company_id = company_1
 
     def test_search_project_root_id(self):
+        self.env['project.project'].search([]).allow_milestones = True
```

Comment:
I am not sure it is really useful, the test seems to use the project created and maybe the one of the parent task and that's it, no? 🧐

Reply:
The problem is that by default, without demo data, all other projects will have `allow_milestones` to False, so the assert at line 225 will fail